MAUI/Segmented-Control/selection.md

```diff
@@ -0,0 +1,423 @@
+---
+layout: post
+title: Selection in .NET MAUI Segmented control | Syncfusion
```

Comment:
Segment item selection in .NET MAUI Segmented control

Reply:
changed title